framework/replay_buffer.py

```diff
@@ -115,14 +115,13 @@ def sample_trajectories(
             for start in indices
         ]
         logging.debug(f"trajectory length: {traj_len}, ")
-
         return trajectories[:mb_size], trajectories[mb_size:]
 
     def _get_samples(
         self, batch_inds: np.ndarray, env: Optional[VecNormalize] = None
     ) -> ReplayBufferSampleHF:
         # Sample randomly the env idx
-        env_indices = np.random.randint(0, high=self.n_envs, size=(len(batch_inds),))
+        env_indices = np.random.randint(0, high=self.n_envs)
```

Comment:
I changed this to only one env, because we else generate trajectories which are not comparable for humans.